_layouts/home.html

```diff
@@ -9,10 +9,10 @@
         <strong class="main-cta__strong">Apache OpenWhisk</strong> is a
         <a class="main-cta__link"
            href="{{ site.github.url }}/serverless">serverless</a>,
-        <strong class="main-cta__strong">open-source</strong> cloud platform
+        <strong class="main-cta__strong">open source</strong> cloud platform
```

Comment:
See https://github.com/openwhisk/openwhisk.github.io/issues/120#issuecomment-266039624 for a pedantic argument about the "right way" to approach hyphenation.